client/src/components/Workflow/WorkflowRunButton.vue

```diff
@@ -3,6 +3,7 @@
         v-b-tooltip.hover.bottom
         :title="title | localize"
         :data-workflow-run="id"
+        :disabled="clicked"
```

Comment:
That doesn't look like it's the right button ?

Reply:
This is the one we need to prevent being clicked while it is busy: https://github.com/jmchilton/galaxy/blob/0e142414cd2d38be75e3b82e30e1d0c1c73b7c23/client/src/components/Workflow/Run/WorkflowRunFormSimple.vue#L6

Reply:
Does it make sense to move this to a separate component, say `GButtonOnce`, to make this reusable?